services/api-server/src/simcore_service_api_server/core/settings.py

```diff
@@ -58,10 +58,15 @@ class BasicSettings(BaseCustomSettings, MixinLoggingSettings):
     )
 
     # LOGGING
-    LOG_LEVEL: LogLevel = Field(
-        default=LogLevel.INFO.value,
```

Comment:
I think the issue was the `.value` at the end. Did you try removing that one?

Reply:
This is wrong. I missed it. I thought i corrected it everywhere